cmd/feedback/feedback.go

```diff
@@ -171,7 +209,8 @@ func NewFeedbackCommand(clients *shared.ClientFactory) *cobra.Command {
 		Short:   "Share feedback about your experience or project",
 		Long:    "Help us make the Slack Platform better with your feedback",
 		Example: style.ExampleCommandsf([]style.ExampleCommand{
-			{Command: "feedback", Meaning: "Open a feedback survey in your browser"},
+			{Command: "feedback", Meaning: "Choose to give feedback on part of the Slack Platform"},
+			{Command: "feedback --name slack-cli-feedback", Meaning: "Give feedback on the Slack CLI"},
```

Comment:
```suggestion
			{Command: "feedback --name slack-cli", Meaning: "Give feedback on the Slack CLI"},
```

AFAICT the suggested changes in comments above catch all changes from "slack-cli-feedback" to "slack-cli" if this is a change we're wanting to make!

Reply:
Thanks! Commits [f5b343f](https://github.com/slackapi/slack-cli/pull/66/commits/f5b343fbb3a425e34445a80b0e321e3af5c0679f) and [48802d9](https://github.com/slackapi/slack-cli/pull/66/commits/48802d92f7cc269561cda48e1d5a94e1119dfed6) change the `--name slack-cli` and update the 2 output references!